sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -16,11 +16,9 @@ Azure Cognitive Services Form Recognizer is a cloud service that uses machine le
 Install the Azure Form Recognizer client library for .NET with [NuGet][nuget]:
 
 ```PowerShell
-dotnet add package Azure.AI.FormRecognizer --version 3.0.0
+dotnet add package Azure.AI.FormRecognizer
```

Comment:
Do we not need to mention the releasing version here?

Reply:
At least in .NET, after we release a GA, we want to keep the versions out of the Readme, so the user can get the stable package by default. If they wasn't a new version, they need to look into NuGet to pick one